sql/sys_vars.cc

```diff
@@ -4686,7 +4686,7 @@ static Sys_var_version Sys_version(
 static char *server_version_suffix_ptr;
 static Sys_var_charptr Sys_version_suffix(
        "version_suffix", "version_suffix",
-       GLOBAL_VAR(server_version_suffix_ptr), NO_CMD_LINE,
+       GLOBAL_VAR(server_version_suffix_ptr), CMD_LINE(REQUIRED_ARG),
```

Comment:
What is this change needed for?

Reply:
This allows to create `keyring_vault` MTR test suite combinations (`plugin/keyring_vault/tests/mtr/combinations `)
```
[mount-v1]
version-suffix=mount-v1

[mount-v2]
version-suffix=mount-v2
...
```
by passing additional info to the server via not very meaningful variable `version-suffix`.
This variable `@@version_suffix` is parsed inside `plugin/keyring_vault/tests/mtr/parse_combination.inc`